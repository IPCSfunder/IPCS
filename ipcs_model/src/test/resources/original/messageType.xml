<?xml version='1.0' encoding='UTF-8'?>
<dataset>
    <message_type message_type_objid="1" type_name="MESSAGE" description="message"></message_type>
    <message_type message_type_objid="2" type_name="VIDEO" description="video"></message_type>
</dataset>