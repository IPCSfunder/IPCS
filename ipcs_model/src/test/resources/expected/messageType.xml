<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<message_type type_name="MESSAGE" description="message"></message_type>
	<message_type type_name="VIDEO" description="video"></message_type>
	<message_type type_name="IMAGE" description="image"></message_type>
</dataset>